src/System.Runtime.Serialization.Xml/tests/DataContractSerializer.cs

```diff
@@ -3062,8 +3062,23 @@ public static void DCS_ResolveNameVariationTest()
         SerializationTestTypes.ComparisonHelper.CompareRecursively(instance, result);
     }
 
+    [Fact]
+    public static void DCS_BasicRoundtripDCRDefaultCollections()
+    {
+        SerializationTestTypes.DefaultCollections defaultCollections = new SerializationTestTypes.DefaultCollections();
```

Comment:
Since many classes which are going to added into SerializationTestTypes contains same class name with the classes in SerializationTypes. If add "using SerializationTestTypes"  at the top of the file, it will cause some conflicts. If we  change the class names ne by one in the later added task, it maybe a little unnecessary.  I think remove "using SerializationTestTypes" is better.